CMakeLists.txt

```diff
@@ -1,34 +1,67 @@
 cmake_minimum_required(VERSION 3.5)
-project(libethercat)
+project(libethercat VERSION 0.4.0)
```

Comment:
last version at DLR was 0.5.1 so we should go for that (or anything greater). I know configure.ac/project.properties is not up to date because our CI automatically generates the version form the tag/branch name (see conanfile.py).

Reply:
Ah, I see. I can fix that, but you've already merged it. Should I open a new PR or should we leave it as is?

Reply:
I fixed that in reading the version from project.properties. Thanks